great_expectations/core/partitioners.py

```diff
@@ -36,6 +39,7 @@ class PartitionerDatetimePart(pydantic.BaseModel):
     column_name: str
     sort_ascending: bool = True
     method_name: Literal["partition_on_date_parts"] = "partition_on_date_parts"
+    param_names: List[str] = []
```

Comment:
these Partitioners are not supported will be removed soon; this attribute is a placeholder for mypy.

Reply:
found a cleaner solution that doesn't require changing these classes [d2cd4ae](https://github.com/great-expectations/great_expectations/pull/9888/commits/d2cd4aebb46922f7b25dbb8a4a2926b9289bbf36)